app/src/main/res/layout/item_content_cart.xml

```diff
@@ -61,64 +66,64 @@
-                            android:layout_height="wrap_content"
-                            android:padding="@dimen/dp_5"
-                            android:text="@{viewModel.quantity}"
-                            android:textSize="@dimen/sp_18"
-                            />
-                        <View
-                            android:layout_width="@dimen/dp_1"
-                            android:layout_height="match_parent"
-                            android:layout_margin="@dimen/dp_5"
-                            android:background="@color/cardview_dark_background"
-                            />
-                        <ImageButton
-                            android:id="@+id/button_up_quantity"
-                            android:layout_width="wrap_content"
-                            android:layout_height="wrap_content"
-                            android:layout_gravity="center"
-                            android:background="@android:color/transparent"
-                            android:onClick="@{()-> viewModel.upQuantity()}"
-                            android:padding="@dimen/dp_5"
-                            android:src="@drawable/ic_add_black"
-                            />
-                    </LinearLayout>
+                        android:gravity="center"
+                        android:paddingTop="@dimen/dp_5"
+                        android:text="@{viewModel.orderTime}"
+                        android:textColor="@android:color/black"
+                        android:textSize="@dimen/sp_15"
+                        android:textStyle="italic"
+                        />
                 </LinearLayout>
-                <ImageButton
-                    android:id="@+id/button_delete_product"
-                    android:layout_width="wrap_content"
+                <ImageView
+                    android:layout_width="match_parent"
                     android:layout_height="wrap_content"
                     android:layout_gravity="center"
-                    android:layout_marginRight="@dimen/dp_20"
-                    android:background="@android:color/transparent"
-                    android:onClick="@{()-> viewModel.deleteProduct()}"
-                    app:srcCompat="@drawable/ic_delete"
+                    android:layout_weight="1.15"
+                    android:onClick="@{()-> viewModel.clickIconWarning()}"
+                    android:src="@drawable/ic_warning"
+                    android:visibility="@{viewModel.productTimeOut ? View.VISIBLE : View.GONE}"
+                    tools:ignore="ContentDescription"
                     />
+                <LinearLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"
+                    android:layout_weight="1"
+                    android:gravity="center"
+                    android:orientation="vertical"
+                    android:paddingStart="@dimen/dp_10"
```

Comment:
WARNING: [RtlSymmetry] When you define `paddingStart` you should probably also define `paddingEnd` for right-to-left symmetry
 If you specify padding or margin on the left side of a layout, you should probably also specify padding on the right side (and vice versa) for right-to-left layout symmetry.